src/components/pages/CreateNote.js

```diff
@@ -0,0 +1,95 @@
+import PropTypes from 'prop-types'
+import React, { useState } from 'react'
+import { useHistory, useParams } from 'react-router-dom'
+import { v4 as uuidv4 } from 'uuid'
+import ButtonPrimary from '../form/Button'
+import InputText from '../form/InputText'
+import TextEditor from '../form/TextEditor'
+import styled from 'styled-components/macro'
+
+CreateNote.propTypes = {
+  setTrips: PropTypes.func,
+  trips: PropTypes.array,
+}
+
+export default function CreateNote({ setTrips, trips }) {
+  const uniqueId = uuidv4()
+  const [editorContent, setEditorContent] = useState('')
+  const params = useParams()
+  const trip = trips.find(trip => trip.id === params.id)
+  const notes = trip.notes || []
+  const history = useHistory()
+
+  return (
+    <WrapperStyled>
+      <h1>Neue Notiz</h1>
+      <Form onSubmit={handleSubmit}>
+        <InputText
+          type="text"
+          name="title"
+          placeholder="Titel eingeben ..."
+          isRequired={true}
+        ></InputText>
+        <TextEditor
+          value={editorContent}
+          handleEditorChange={handleEditorChange}
+        />
+        <div className="center">
+          <ButtonPrimary
+            label={'Notiz speichern'}
+            content="Notiz speichern"
+          ></ButtonPrimary>
+        </div>
+      </Form>
+    </WrapperStyled>
+  )
+
+  function handleEditorChange(editorContent) {
+    setEditorContent(editorContent)
+  }
```

Comment:
`handleEditorChange` is just a wrapper. Maybe delete it and use `setEditorContent` directly?